m4/gnulib-cache.m4

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2002-2025 Free Software Foundation, Inc.
+# Copyright (C) 2002-2023 Free Software Foundation, Inc.
```

Comment:
This change isn't right. I created it with a gnulib-tool last modified this year.